op-batcher/batcher/driver.go

```diff
@@ -137,6 +137,19 @@ func (l *BatchSubmitter) StartBatchSubmitting() error {
 	l.clearState(l.shutdownCtx)
 	l.lastStoredBlock = eth.BlockID{}
 
+	// Check L2 genesis time
+	genesisTime := time.Unix(int64(l.RollupConfig.Genesis.L2Time), 0)
```

Comment:
It could be nice to have a ticker doing here to periodically log the current time and how much longer its expected to wait, this is just nice devex for chain operators

Reply:
thanks for the suggest, please take another look.